### ISSUE TITLE
Resource registry_key { should_not exist } unavailable

### ISSUE THREAD

**Issue by larryebaum** [inspec/inspec#322]
The following code fails as it isn't supported; it is necessary as in many cases we want to ensure that either a key doesn't exist but in the case that if it does, that it matches a particular value. The former isn't possible whereas the latter is.

```
describe registry_key('path\to\key') do
  it { should_not exist }
end
```

**Comment by chris-rock** [comment]
We should have an integration test for this bug as well. Thanks @larryebaum for reporting this.

**Comment by chris-rock** [comment]
solved with #364 please reopen if this stays a problem after this release
